src/api/downloadFile.js

```diff
@@ -0,0 +1,27 @@
+/* @flow */
+import { Platform, CameraRoll } from 'react-native';
+import RNFetchBlob from 'react-native-fetch-blob';
+
+import { getAuthHeader } from '../utils/url';
+import userAgent from '../utils/userAgent';
+import { Auth } from '../types';
+
+export default (url: string, auth: Auth) => {
+  if (Platform.OS === 'android') {
```

Comment:
You can use any other file as platform specific though.
So `downloadFile.ios.js` and `downloadFile.android.js`